assets/src/design-system/components/search/input/index.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useMemo, forwardRef } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import {
+  Input,
+  InputContainer,
+  ClearIcon,
+  ChevronIcon,
+  ChevronDecoration,
+  SearchDecoration,
+  SearchIcon,
+  ClearButton,
+} from './components';
+
+const SearchInput = (
+  {
+    ariaClearLabel,
+    clearId,
+    disabled,
+    handleClearInputValue,
+    handleTabClearButton,
+    inputValue,
+    isOpen,
+    listId,
+    ...rest
+  },
+  ref
+) => {
+  // Avoid conditional rendering in this input because it causes rerendering which if the keyboard is getting used to focus the input will remove the visible focus style.
```

Comment:
Should we trim this down?
```suggestion
  // Avoid conditional rendering in this input because rerendering will remove the focus styling
```

Reply:
thanks!